spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,17 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET #index' do
+    before(:example) { get '/posts' } # get(:index)
+
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+    it "renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+    it 'includes correct placeholder text' do
+      expect(response.body).to include('Show posts for a given user:')
+    end
```

Comment:
- Good job implementing the Post `#index` action for this test 👍🏽. Kindly add a test for the Post `#show` action to complete the test requirement 😉.